Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBSDKKeychain.m

```diff
@@ -67,8 +67,7 @@ + (NSString *)retrieveTokenWithKey:(NSString *)key {
 }
 
 + (NSArray<NSString *> *)retrieveAllTokenIds {
-  NSMutableDictionary<id, id> *query = [DBSDKKeychain
-      queryWithDict:@{(id)kSecReturnAttributes : (id)kCFBooleanTrue, (id)kSecMatchLimit : (id)kSecMatchLimitAll}];
+  NSMutableDictionary<id, id> *query = [DBSDKKeychain queryWithDict:@{(id)kSecReturnAttributes : (id)kCFBooleanTrue, (id)kSecMatchLimit : (id)kSecMatchLimitAll}];
```

Comment:
These indentations are automatically set by our clang code formatter, so it will probably be reset to what it currently is again :)